vroom/bazel.vroom

```diff
@@ -9,7 +9,6 @@ The main command this plugin provides is :Bazel, which drops to a shell and
 invokes bazel in the foreground, leaving output visible with a "Press ENTER"
 prompt until explicitly dismissed.
 
-  :Bazel info
+  :Bazel info <CR>
```

Comment:
For some reason,  vroom always just hung unless I added this. I couldn't get the input functionality to recognize `> <CR>` with neovim. This seems to work though.

Reply:
And by hang, I mean it would pause waiting for ENTER.